api/dataset_test.go

```diff
@@ -1004,6 +1004,152 @@ func TestPutDatasetReturnsError(t *testing.T) {
 	})
 }
 
+func TestDeleteDatasetReturnsSuccessfully(t *testing.T) {
+	t.Parallel()
+	Convey("", t, func() {
```

Comment:
Missing test description

Reply:
Added descriptions for all tests that were missing them